openedx/core/lib/api/paginators.py

```diff
@@ -3,6 +3,31 @@
 from django.http import Http404
 from django.core.paginator import Paginator, InvalidPage
 
+from rest_framework.response import Response
+from rest_framework import pagination
+
+
+class DefaultPagination(pagination.PageNumberPagination):
```

Comment:
Can we merge this with the TeamAPIPagination and then remove the latter? Teams really should use the same pagination scheme as everything else, and we can fix the clients if necessary.